plugins/kosync.koplugin/main.lua

```diff
@@ -507,6 +542,28 @@ function KOSync:getLastProgress()
     end
 end
 
+function KOSync:getDocumentDigest()
+    if self.kosync_checksum_method == CHECKSUM_METHOD.FILENAME then
+        return self:getFileNameDigest()
+    else
+        return self:getFileDigest()
+    end
+end
+
+function KOSync:getFileDigest()
+    return self.view.document:fastDigest()
+end
+
+function KOSync:getFileNameDigest()
+    local file = self.ui.document.file
+    if not file then return end
+
+    local filePath, fileName = util.splitFilePathName(file) -- luacheck: no unused
+    if not fileName then return end
+
+    return md5(fileName)
```

Comment:
(Also, our style guide reserves camelCase to function names, so these variables better be called `filename` or `file_name`.)

Reply:
Thanks for catching this. Updated to be file_path and file_name per style guide.